src/draw/vg_lite/lv_draw_vg_lite.c

```diff
@@ -134,21 +135,15 @@ static void draw_execute(lv_draw_vg_lite_unit_t * u)
             break;
     }
 
-#if LV_USE_PARALLEL_DRAW_DEBUG
-    /* Layers manage it for themselves. */
-    if(t->type != LV_DRAW_TASK_TYPE_LAYER) {
-    }
-#endif
-
-    LV_VG_LITE_CHECK_ERROR(vg_lite_finish());
+    lv_vg_lite_flush(draw_unit);
```

Comment:
What if `flush_count == 3` and there are no more draw tasks. What will trigger the flushing of these 3 comamnds?